tools/ci/checks/get_applicable_changes.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Please do this in Python. It will probably be shorter and easier to see whats going on.